### PR TITLE
UX Improvements

### PR DESCRIPTION
**Pull request by jannikrh**

### CODE REVIEW

**Comment by jannikrh** [comment]
[Improve focus nodes](https://trello.com/c/2jzNd3ig/27-improve-focus-nodes)

File: lib/login/view/login_form.dart [hunk 1]
```diff
@@ -53,13 +55,18 @@ class LoginForm extends StatelessWidget {
...
+  _EmailInput(this.nextFocusNode);
+
   @override
   Widget build(BuildContext context) {
     return BlocBuilder<LoginCubit, LoginState>(
       buildWhen: (previous, current) => previous.email != current.email,
       builder: (context, state) {
         return TextField(
           key: const Key('loginForm_emailInput_textField'),
+          autocorrect: true,
```
Comment (NotAnsgar) [inline lib/login/view/login_form.dart:69]: I'm not sure what this line does. So if this line is not necessary remove this line, else keep it.

**Review by NotAnsgar — APPROVED** [review]
Overall this PR is fine. Im not sure what the autocorrect line does.

File: lib/login/view/login_form.dart (hunk 1 above)
Comment (jannikrh) [reply lib/login/view/login_form.dart:69]: This enables autocorrection on smartphones while entering the mail address. This leads to the intended behavior that your own typical mails are suggested by the smartphone. The value true is also default, the extra configuration clarifies the (intentional) requirement.

**Pull request merged**